webapp/app/static/jsx/components/AgnosticTable.jsx

```diff
@@ -28,12 +28,12 @@ export default function AgnosticTable({ toIgnore = new Set(), keyValues, aliases
                     }
                     return (
                         <tr className="detail-row" key={idx}>
-                            <td>{k} 
-                            {hovers && hovers[ogK] && <MyTooltip tooltipText={hovers[ogK]} />}
+                            <td>{k}
+                                {hovers && hovers[ogK] && <MyTooltip tooltipText={hovers[ogK]} />}
```

Comment:
Now that there's no spacing between the key and value, it's hard to read that the value corresponds to the key (for example, the heuristic is 'linked transaction' in this case). can you add a ':' at the end of this line, line 32? after the curly bracket? 
![image](https://user-images.githubusercontent.com/44105002/149239907-b668ce2b-119e-4c35-bf2d-c34f0f2309f7.png)